Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampaignIncidentsInfo.py

```diff
@@ -39,6 +39,12 @@ def update_incident_with_required_keys(incidents: List, required_keys: List):
         :param required_keys: keys need to be updated
 
     """
+    for incident in incidents:
+        res = demisto.executeCommand('GetIncidentsByQuery', {'query': "id:({})".format(incident['id'])})
+        # If an incident is deleted, remove it from the list to avoid a key error
+        if res[0]['Contents'] == '[]':
```

Comment:
is this safe enough? (avoiding `IndexError`s)

Reply:
Fixed